modules/client-common/src/main/java/org/apache/ignite/internal/client/proto/ClientMessagePacker.java

```diff
@@ -36,637 +34,748 @@
+            packDouble((double) val);
+            
+            return;
         }
-
+        
         if (val instanceof UUID) {
-            return packUuid((UUID) val);
+            packUuid((UUID) val);
+            
+            return;
         }
-
+        
         if (val instanceof String) {
-            return (ClientMessagePacker) packString((String) val);
+            packString((String) val);
+            
+            return;
         }
-
+        
         if (val instanceof byte[]) {
             byte[] bytes = (byte[]) val;
             packBinaryHeader(bytes.length);
             writePayload(bytes);
-
-            return this;
+            
+            return;
         }
-
+        
         if (val instanceof BigDecimal) {
-            return packDecimal((BigDecimal) val);
+            packDecimal((BigDecimal) val);
+            
+            return;
         }
-
+        
         if (val instanceof BigInteger) {
-            return packNumber((BigInteger) val);
+            packNumber((BigInteger) val);
+            
+            return;
         }
-
+        
         if (val instanceof BitSet) {
-            return packBitSet((BitSet) val);
+            packBitSet((BitSet) val);
+            
+            return;
         }
-
+        
         if (val instanceof LocalDate) {
-            return packDate((LocalDate) val);
+            packDate((LocalDate) val);
+            
+            return;
         }
-
+        
         if (val instanceof LocalTime) {
-            return packTime((LocalTime) val);
+            packTime((LocalTime) val);
+            
+            return;
         }
-
+        
         if (val instanceof LocalDateTime) {
-            return packDateTime((LocalDateTime) val);
+            packDateTime((LocalDateTime) val);
+            
+            return;
         }
-
+        
         if (val instanceof Instant) {
-            return packTimestamp((Instant) val);
+            packTimestamp((Instant) val);
+            
+            return;
         }
-
-        throw new UnsupportedOperationException("Unsupported type, can't serialize: " + val.getClass());
+        
+        throw new UnsupportedOperationException(
+                "Unsupported type, can't serialize: " + val.getClass());
     }
-
+    
     /**
      * Packs an array of different objects.
      *
      * @param args Object array.
-     * @return This instance.
      * @throws UnsupportedOperationException in case of unknown type.
      */
-    public ClientMessagePacker packObjectArray(Object[] args) {
+    public void packObjectArray(Object[] args) {
         assert !closed : "Packer is closed";
-
+        
         if (args == null) {
             packNil();
-
-            return this;
+            
+            return;
         }
-
+        
         packArrayHeader(args.length);
-
+        
         for (Object arg : args) {
             if (arg == null) {
                 packNil();
-
+                
```

Comment:
Maybe you should tweak your IDE a bit to remove trailing whitespaces :)

Reply:
I have this enabled, but I've used auto-reformat to conform with the new code style and that's where those spaces come from.

Fixed.